src/android/app/src/main/java/io/github/lime3ds/android/features/settings/ui/SettingsFragmentPresenter.kt

```diff
@@ -873,6 +902,232 @@ class SettingsFragmentPresenter(private val fragmentView: SettingsFragmentView)
         }
     }
 
+    private fun addLayoutSettings(sl: ArrayList<SettingsItem>) {
```

Comment:
At the moment the Layout menu is quite cluttered. To combat this I would recommend breaking the Landscape and Portrait custom layout settings into two sub-menus rather than grouping them together.

Reply:
I see that you changed the strings in the layout menu to simply say "X-Position" and "Y-Position" etc, but for now that means that the bottom screen and top screen have the same labels. Were you thinking that when I move these into sub-menus I should use a heading/title to separate the top and bottom screen layout options?

Reply:
That's what I did, I think it's nice.